### PR TITLE
test

### PR DESCRIPTION
**Pull request by 2yunj007**
test

### CODE REVIEW

**Review by megar0829 — COMMENTED** [review]
comment

**Review by wjstp — CHANGES_REQUESTED** [review]
request change?

**Review by wjstp — APPROVED** [review]
Approve

**Review by megar0829 — DISMISSED** [review]
approve

**Comment by wjstp** [comment]
mergemerge

**Review by wjstp — COMMENTED** [review]
comment로 변경

**Comment by wjstp** [comment]
> request change?

**Comment by wjstp** [comment]
> request change?

**Review by wjstp — DISMISSED** [review]
ok

**Review by megar0829 — APPROVED** [review]

File: test11.py [hunk 1]
```diff
@@ -41,4 +41,6 @@
         # 기업 정보 페이지에 있는 로고 이미지 가져오기
         company_url = companys[i]['href']
         if company_url[0] == '/':
-            company_url = f"https://www.jobkorea.co.kr/{company_url}"
```
Comment (megar0829) [inline test11.py:44]: dd

File: test11.py [hunk 2]
```diff
@@ -41,4 +41,6 @@
...
\ No newline at end of file
+            company_url = f"https://www.jobkorea.co.kr/{company_url}"
+
```
Comment (wjstp) [inline test11.py]: test

**Comment by megar0829** [comment]
> request change?

 너 뭐야

**Review by wjstp — APPROVED** [review]

**Review by wjstp — APPROVED** [review]

**Review by wjstp — APPROVED** [review]

**Review by megar0829 — APPROVED** [review]

**Comment by 2yunj007** [comment]
읭

**Pull request closed without merging**